static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -94,21 +100,25 @@ function Breadcrumbs({frames, startTimestampMs}: Props) {
 
   return (
     <FluidHeight>
-      <TabItemContainer>
+      <BreadcrumbFilters frames={frames} {...filterProps} />
+      <TabItemContainer data-test-id="replay-details-breadcrumbs-tab">
         {frames ? (
           <AutoSizer onResize={updateList}>
             {({height, width}) => (
               <ReactVirtualizedList
                 deferredMeasurementCache={cache}
                 height={height}
                 noRowsRenderer={() => (
-                  <NoRowRenderer unfilteredItems={frames} clearSearchTerm={() => {}}>
+                  <NoRowRenderer
+                    unfilteredItems={items}
+                    clearSearchTerm={clearSearchTerm}
+                  >
                     {t('No breadcrumbs recorded')}
                   </NoRowRenderer>
                 )}
                 overscanRowCount={5}
                 ref={listRef}
-                rowCount={frames.length}
+                rowCount={items.length}
```

Comment:
This is correct to be `items`, because it's the count of what we're showing in the list.